src/distilabel/steps/task/data.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2023-present, Argilla, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import TYPE_CHECKING, Any, Dict, List, Optional
+
+from typing_extensions import override
+
+from distilabel.llm.base import LLM
+from distilabel.steps.base import (
+    GeneratorStep,
+)
+from distilabel.steps.task.base import _Task
+
+if TYPE_CHECKING:
+    from distilabel.steps.task.typing import ChatType
+    from distilabel.steps.typing import GeneratorStepOutput
+
+
+class DataTask(_Task, GeneratorStep):
```

Comment:
Hey, this doesn't need to inherit from `_Task`, it only needs to inherit from `GeneratorStep`.

Reply:
I would call this class `LoadData` or `YieldData` or `GenerateData`, maybe `LoadData` is the best one.